client/app/lib/util/makeHttpClient.coffee

```diff
@@ -0,0 +1,50 @@
+{ assign } = require 'lodash'
+axios = require 'axios'
```

Comment:
It's an easy, `http-client` library that returns promises for each request, and also has the capability of creating individual instances that is bound to specific requests. for example creating a client with a base url of `/payment` after that each request for that instance will be prefixed with that url, more info is [here](https://github.com/mzabriskie/axios/#creating-an-instance)